prowler/__main__.py

```diff
@@ -112,6 +113,11 @@ def prowler():
         and not checks_folder
     )
 
+    if args.no_color:
+        colorama_init(strip=True)
+    else:
+        colorama_init(strip=False)
```

Comment:
```suggestion
```
Is this needed?

Reply:
There's a good chance it's not strictly required. However, as the default is 'None' with an OS-dependent behavior I thought it was wise to keep.  Reviewing the OS-dependent behavior, I agree, this should be remove.

Reply:
fixed.

Reply:
Thanks! I'll add to your PR info about this flag inside the docs, thanks for your contribution!